google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITKmsTest.java

```diff
@@ -186,46 +179,6 @@ public void testGetBlobKmsKeyNameField() {
     assertNull(remoteBlob.getContentType());
   }
 
-  @Test(timeout = 5000)
-  public void testListBlobsKmsKeySelectedFields() throws InterruptedException {
```

Comment:
This test was "testing" if an object with a kms key would have said kms key populated in a list response. Field masks and mapping are verified elsewhere, deleting here.